etl/steps/data/meadow/un/2024-02-14/sdgs_urbanization.py

```diff
@@ -0,0 +1,39 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("un_sdg")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["un_sdg"].reset_index()
+
+    urbanization_tb = tb[(tb["indicator"] == "11.7.1") | (tb["indicator"] == "11.2.1")]
+    cols = ["seriesdescription", "country", "year", "value", "cities"]
+    urbanization_tb = urbanization_tb[cols]
+    # Convert the seriesdescription to string to avoid using categorical data
+    urbanization_tb["seriesdescription"] = urbanization_tb["seriesdescription"].astype(str)
```

Comment:
Nit - this won't help you, calling `ds_meadow.save()` converts string to categorical type anyway (to make the dataset smaller on disk).